vite.config.mjs

```diff
@@ -0,0 +1,216 @@
+import react from '@vitejs/plugin-react-swc'
+import * as fs from 'fs'
+import { CID } from 'multiformats/cid'
+import * as raw from 'multiformats/codecs/raw'
+import { sha256 } from 'multiformats/hashes/sha2'
+import { dirname, resolve } from 'path'
+import * as path from 'path'
+import * as ssri from 'ssri'
+import { fileURLToPath } from 'url'
+import { defineConfig, loadEnv } from 'vite'
```

Comment:
```suggestion
import { defineConfig } from 'vite'
```

Reply:
Done in [8c397de](https://github.com/shapeshift/web/pull/8924/commits/8c397de36624ea090d70e8ab38932ab07a7e2e84)